electrochemistry.ttl

```diff
@@ -3946,13 +4265,12 @@ Cl2, Hg | Hg2SO4, and Hg | HgO, can be used as reference electrodes in aqueous s
 :electrochemistry_9da958fc_f76d_4654_8a78_99b5f98c118c rdf:type owl:Class ;
                                                        rdfs:subClassOf emmo:EMMO_b9522e56_1fac_4766_97e6_428605fabd3e ,
                                                                        [ rdf:type owl:Restriction ;
-                                                                         owl:onProperty emmo:EMMO_dba27ca1_33c9_4443_a912_1519ce4c39ec ;
+                                                                         owl:onProperty :electrochemistry_3bd08946_4e81_455d_9fca_dc7a5ead9315 ;
```

Comment:
However, here, it is preferred to use `hasElectrolyte`. I would tend to favor using `hasConstituient `as much as possible, without needing to create additional object properties. `hasConstituent` something of `@type: Electrolyte` is possible in linked data formats, removing the need of creating a very specific object property.

Reply:
see https://github.com/emmo-repo/domain-electrochemistry/pull/61#discussion_r1339961238